style/ruby/README.md

```diff
@@ -27,6 +27,7 @@ Ruby
 * Prefer `if` over `unless`.
 * Use `_` for unused block parameters.
 * Prefix unused variables or parameters with underscore (`_`).
+* Use a leading underscore when defining instance variables for memoization.
```

Comment:
I personally lean towards `Use`, but if others think `Prefer`is more appropriate I can update it.